seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/AbstractWriteStrategy.java

```diff
@@ -328,6 +329,24 @@ public String getTargetLocation(@NonNull String seaTunnelFilePath) {
         return tmpPath.replaceAll(BaseSinkConfig.NON_PARTITION + Matcher.quoteReplacement(File.separator), "");
     }
 
+    /**
+     * Delete temporary files not caused by checkpoint.
+     * @param tmpFilePaths temporary files.
+     * @return void
+     */
+    public void deleteTmpFileWithNoCheckPoint(Set<String> tmpFilePaths){
+        try {
+            if (tmpFilePaths != null && tmpFilePaths.size() > 0) {
+                for (String filePath : tmpFilePaths){
+                    fileSystemUtils.deleteFile(filePath);
+                    log.info("delete no commit file:{}", filePath);
```

Comment:
What's the meaning of 'no commit file'? Why not use the 'temporary file ' or 'uncommitted file'?

Reply:
complete modify.